templates/frontend/inputs/measure.php

```diff
@@ -84,7 +83,7 @@ class="<?php echo esc_attr( $input_classes ); ?>"
 				<?php
 				// Add input extra attributes
 				foreach ( $input_attr as $key => $val ) {
-					echo $key . '="' . $val . '"';
+					echo sprintf( '%s="%s"', esc_html( $key ), esc_html( $val ) );
 				}
```

Comment:
I think, using `esc_attr` instead of `esc_html` would be better here (valid other inputs too)